library/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
     id 'bio.terra.pfb.java-library-conventions'
     id 'com.github.davidmc24.gradle.plugin.avro' version '1.9.1'
     id 'me.champeau.jmh' version '0.7.3'
+    id 'com.google.cloud.artifactregistry.gradle-plugin' version '2.1.5'
```

Comment:
can the `id 'com.jfrog.artifactory'` plugin be removed now, up at line 4?

Reply:
Ah right I meant to do that and forgot, thanks